apps/android/app/src/main/java/chat/simplex/app/views/chatlist/UserPicker.kt

```diff
@@ -162,15 +169,17 @@ fun UserProfilePickerItem(u: User, unreadCount: Int = 0, onLongClick: () -> Unit
       )
     }
     if (u.activeUser) {
-      Icon(Icons.Filled.Done, null, Modifier.size(20.dp), tint = MaterialTheme.colors.onBackground)
+        Icon(Icons.Filled.Done, null, Modifier.size(20.dp), tint = MaterialTheme.colors.onBackground)
+    } else if (u.hidden) {
+        Icon(Icons.Filled.Lock, null, Modifier.size(20.dp), tint = HighOrLowlight)
```

Comment:
```suggestion
```

this shouldn't show

Reply:
this code is reused with UserPicker and UserProfilesView. But since UserPicker filters users and skips hidden, this condition will never be called in picker. It will be called in UserProfilesView, so it's ok to have it here

Reply:
ok